scripts/sshzobelisk

```diff
@@ -0,0 +1,40 @@
+set -euo pipefail
+
+usage()
+{
+    echo "USAGE: $0 <USERNAME ON ZOBELISK> <USER ID ON ZOBELSIK>"
+    exit 1
+}
+
+if [ $# -ne 2 ]; then
+    usage
+fi
+
+LOCAL_USER=$USER
+SERVER_USERNAME=$1
+SERVER_UID=$2
+LOCAL_REPO=$(realpath $(dirname $BASH_SOURCE)/../)
+ZOBELISK_IP=192.168.37.176
+IP=`hostname -I`
+IP=`echo $IP | head -n1 | awk '{print $1;}'` # `echo $IP | sed 's/ *$//g'`
+echo $IP
+
+sudo systemctl start ssh
```

Comment:
Why does the user need to run an ssh server? They are SSHing _into_ zobelisk

Reply:
Unfortunately, to the best of my knowledge, the only way to share files like this without an ssh server on the users computer is to run an ftps server ,but that is alot. There is now a warning upon running the script and alternative version without the user running the ssh server, but no file sharing.